src/services/mail.service.js

```diff
@@ -22,7 +22,7 @@ class MailService {
 					button: {
 						color: '#33b5e5',
 						text: 'Verify account',
-						link: `http://localhost:3000/api/auth/verify/activate?token=${token}`,
+						link: `http://localhost:3000/api/auth/activate?token=${token}`,
```

Comment:
get the http://localhost:3000 from the environment variable so that it can be chage later

Reply:
Yeah Done.